public/assets/images/arrow_small.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="13.36" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I suggest you create one component in which you return all the arrows that will be needed and we can be exporting them! This is an example to it!

`import React from "react";

export const ArrowPointingRight = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      stroke="%23FFFFFF"
      stroke-width="2"
      stroke-linecap="round"
      stroke-linejoin="round"
    >
      <path d="M5 12h13M12 5l7 7-7 7" />
    </svg>
  );
};

export const ArrowPointing = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="24"
      height="24"
      viewBox="0 0 24 24"
      fill="none"
      stroke="%23FFFFFF"
      stroke-width="2"
      stroke-linecap="round"
      stroke-linejoin="round"
    >
      <path d="M5 12h13M12 5l7 7-7 7" />
    </svg>
  );
};
`